src/REPLMode.jl

```diff
@@ -714,6 +741,28 @@ function do_develop!(ctx::APIOptions, args::PkgArguments, api_opts::APIOptions)
     API.add_or_develop(Context!(ctx), args; collect(api_opts)...)
 end
 
+# registry commands
+function do_registry_add!(ctx::APIOptions, args::PkgArguments, api_opts::APIOptions)
+    # TODO: Should accept a --depot option ?
```

Comment:
From recent discussions I take this as a no? Instead we assume that `depots1()` is a writeable user depot and put it there?

Reply:
I think so, yes.